src/test/java/ashe/AsheTest.java

```diff
@@ -0,0 +1,37 @@
+package ashe;
+
+import edu.njit.jerse.ashe.services.MethodReplacementService;
+import edu.njit.jerse.ashe.utils.JavaCodeParser;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+
+class AsheTest {
+
+    @Test
+    void runAsheTest() throws IOException, InterruptedException {
+        /** In order to run these tests, uncomment the following lines and add your own paths */
```

Comment:
It is a requirement that the test should be able to run out-of-the-box with no user configuration or need to edit paths. These should all be relative paths, anyway, that are known to us (at least, for the particular files that are relevant to this test...).

In other words, until this runs successfully on CI, I don't believe it :)